### PR TITLE
fix null ref

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
### Context

null ref error in preview if no accrediting provider

### Changes proposed in this pull request

null check

### Guidance to review

n/a

### CODE REVIEW

File: shared/Views/Shared/Components/CourseDetails/Default.cshtml [hunk 1]
```diff
@@ -25,9 +25,9 @@
             <h1 class="govuk-heading-xl" role="heading">
                 @Model.Course.Name at @Model.Provider.Name
             
-                @if (!String.IsNullOrEmpty(Model.Course.AccreditingProvider.Name))
+                @if (!String.IsNullOrEmpty(Model.Course.AccreditingProvider?.Name))
                 {
-                    <span class="govuk-caption-xl">Accredited by @Model.Course.AccreditingProvider.Name</span>
+                    <span class="govuk-caption-xl">Accredited by @Model.Course.AccreditingProvider?.Name</span>
```
Comment (fofr) [inline shared/Views/Shared/Components/CourseDetails/Default.cshtml]: Does this need a test?

**Comment by the-daniel-rothig** [comment]
[500 on preview](https://trello.com/c/4VWlZv0T/241-500-on-preview)

File: shared/Views/Shared/Components/CourseDetails/Default.cshtml (hunk 1 above)
Comment (the-daniel-rothig) [reply shared/Views/Shared/Components/CourseDetails/Default.cshtml]: ASP.NET views are pretty untestable :(

Comment (defong) [reply shared/Views/Shared/Components/CourseDetails/Default.cshtml]: I shall refrain from questioning `untestable` statement....

**Pull request merged**